agent/tox-unittests

```diff
@@ -4,6 +4,6 @@ prog="$(basename "${0}")"
 progdir="$(realpath -e $(dirname "${0}"))"
 
 # List of tox environments to run.
-tox_envs="py3-server server"
+tox_envs="agent"
```

Comment:
This is a side comment on GitHub: it decided that you *moved* `server/run-unittests` to `agent/tox-unittests` despite the fact that almost half the lines differ substantially. That's ... intriguing.

Reply:
This file is no longer really useful.  Removing.